runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterConfig.java

```diff
@@ -52,9 +56,22 @@ default boolean showSpecial()
 	@ConfigItem(
 		keyName = "showWhenNoChange",
 		name = "Show hitpoints regen at full hitpoints",
-		description = "Always show the hitpoints regen orb, even if there will be no stat change")
+		description = "Always show the hitpoints regen orb, even if there will be no stat change",
+		position = 3
+	)
 	default boolean showWhenNoChange()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			keyName = "notifyBeforeFullRegen",
+			name = "Notify before full HP regen",
```

Comment:
Name + description is off. Full isn't 2.

Reply:
https://github.com/runelite/runelite/wiki/Code-Conventions